Client/Ecosia/UI/NTP/NTPBookmarkNudgeViewModel.swift

```diff
@@ -0,0 +1,85 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Shared
+import Core
+
+protocol NTPBookmarkNudgeViewDelegate: AnyObject {
```

Comment:
I believe you got the almost "obsessive" approach I've got when it comes to separate files 😇 😅 .
Wondering whether this could be moved into a separate one unless strictly needed here

Reply:
Yep! 😄  Done!